### ISSUE TITLE
Q[option] should also have a detach() when calculating ac_loss

### ISSUE THREAD

**Issue by backpropper** [lweitkamp/option-critic-pytorch#2]
https://github.com/lweitkamp/option-critic-pytorch/blob/0c57da7686f8903ed2d8dded3fae832ee9defd1a/option_critic.py#L241

**Comment by lweitkamp** [comment]
Q is detached in line 230 because it shouldn't be updated in the actor loss in general. Although it is sloppy. I will rewrite the code a bit to make this more clear when I have some time.

**Comment by backpropper** [comment]
Ah I see. In that case, the termination loss has extra detaches.
